mem0/llms/azure_openai.py

```diff
@@ -15,6 +16,10 @@ def __init__(self, config: Optional[BaseLlmConfig] = None):
         if not self.config.model:
             self.config.model = "gpt-4o"
         self.client = AzureOpenAI()
+        
+        if os.getenv("AZURE_OPENAI_API_KEY"):
```

Comment:
`If` statement is not required here. Please change at all the other places as well

Reply:
Ok, understood. 
I have done the changes, can you please review again.